packages/scss/src/components/multiSelect/index.scss

```diff
@@ -61,19 +61,3 @@
 		min-inline-size: 0;
 	}
 }
-
-@layer mods {
-	.multipleSelect-displayer {
-		&.mod-filter {
-			@include displayerFilter;
```

Comment:
We also lose `display: flex` on `.multipleSelect-displayer-filter` that lead to extra spacing under the chip.

https://github.com/user-attachments/assets/0ca3500c-4181-418a-bc36-1b04d42889fc